css/style.css

```diff
@@ -67,5 +69,188 @@ button:hover {
 
 .toggleNav {
   text-align: right;
-  padding-right: 10px;
+  padding: 20px 16px 0 0;
+}
+
+.geometry {
+  display: flex;
+  padding: 0;
+  margin: 0;
+}
+
+.geometry img {
+  flex: 1;
+}
+
+.projects-title h1 {
+  padding: 40px 0 32px 16px;
+  font-family: Playfair Display, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 50px;
+  line-height: 100%;
+  color: #1a2236;
+}
+
+/* Grid start */
+
+.grid-container {
+  display: grid;
```

Comment:
- [ ] This grid CSS layout should be a better fit in the projects class.
- [ ] Flexbox is more appropriate here.